http/src/main/scala/filodb/http/FiloHttpServer.scala

```diff
@@ -49,10 +50,17 @@ class FiloHttpServer(actorSystem: ActorSystem, filoSettings: FilodbSettings) ext
     implicit val system = actorSystem
     implicit val materializer = ActorMaterializer()
     // This is a preliminary implementation of routes. Will be enhanced later
-    val filoRoutes: List[FiloRoute] = List(AdminRoutes,
+    val defaultRoutes: List[FiloRoute] = List(AdminRoutes,
                                            new ClusterApiRoute(clusterProxy),
                                            new HealthRoute(coordinatorRef),
                                            new PrometheusApiRoute(coordinatorRef, settings))
+    val runtimeRoutes = settings.httpRuntimeApiRoutes.asScala.map {
+      Class.forName(_)
+        .getConstructor(classOf[FilodbSettings])
+        .newInstance(filoSettings)
```

Comment:
Document the requirements of the class. i.e. it should (a) implement FiloRoute (b) have a 1 arg constructor taking settings class in.

Reply:
See if you can make it empty arg constructor. FilodbSettings can be gotten from GlobalConfig too.